src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -118,19 +116,23 @@ public boolean Disconnect(String peerId){
     public boolean  DisconnectIncomingConnections(){
 
         boolean ret = false;
+
         if (mServerSocketList != null) {
```

Comment:
@DrJukka As mentioned above this isn't safe. If there are any more examples of this I won't call them out. I'm going to assume that you will search for them.